organisaatio-ui/src/components/Sivut/Ryhmat/Muokkaus/RyhmanMuokkaus.tsx

```diff
@@ -1,178 +1,166 @@
 import * as React from 'react';
-import styles from './RyhmanMuokkaus.module.css';
-import { useContext } from 'react';
-import { KoodistoContext, LanguageContext } from '../../../../contexts/contexts';
-import Input from '@opetushallitus/virkailija-ui-components/Input';
-import PohjaSivu from '../../PohjaSivu/PohjaSivu';
-import Icon from '@iconify/react';
-import homeIcon from '@iconify/icons-fa-solid/home';
-import Button from '@opetushallitus/virkailija-ui-components/Button';
 import { useState } from 'react';
-import Select from '@opetushallitus/virkailija-ui-components/Select';
+import { RouteComponentProps } from 'react-router-dom';
 import Spin from '@opetushallitus/virkailija-ui-components/Spin';
 import { useEffect } from 'react';
-import Axios from 'axios';
+import Axios, { AxiosResponse } from 'axios';
+import { Ryhma, SelectOptionType } from '../../../../types/types';
+import { useLanguagedInput } from '../../../../customHooks/CustomHooks';
+import { ActionMeta, ValueType } from 'react-select';
+import MuokkausLomake from './MuokkausLomake';
+import { useContext } from 'react';
+import { LanguageContext } from '../../../../contexts/contexts';
+
+export type RyhmanMuokausProps = {
+    oid: string;
+};
 
-type Props = {
-    match: any;
+export type OrganisaatioPutResponseType = {
+    status: string;
+    organisaatio: Ryhma;
 };
 
-export default function RyhmanMuokkaus(props: Props) {
-    const { i18n, language } = useContext(LanguageContext);
-    const { ryhmaTyypitKoodisto, kayttoRyhmatKoodisto } = useContext(KoodistoContext);
-    // const [isKaikkiValittu, setIsKaikkiValittu ] = useState(KAIKKIVALITTU);
-    const [ryhma, setRyhma] = useState<undefined | any>();
-    //const [isModaliAuki, setIsModaaliAuki ] = useState(false);
-    const {
-        match: { params },
-    } = props;
+export const currentDateToStr = () => new Date().toISOString().split('T')[0];
+
+export const putRyhma = async (ryhma: Ryhma) => await Axios.put(`/organisaatio/organisaatio/v4/${ryhma.oid}`, ryhma);
```

Comment:
You mentioned this already before. I myself like the calls and the urls are clearly visible where they are called, not in some separate file. I might be wrong but l like it that way. Do you want this to be changed so that they all are in a separate file?

Reply:
Again, I do not _want_ anything. I just foresee some problems with the approach